cloudstate/cloudstate.go

```diff
@@ -74,6 +78,17 @@ func (cs *CloudState) RegisterCRDT(entity *crdt.Entity, config protocol.Descript
 	return nil
 }
 
+// RegisterCRDT registers a CRDT entity.
```

Comment:
Comment is for CRDT entity

Reply:
Great catch, thanks for reading Go code!
Fixed with: https://github.com/cloudstateio/go-support/pull/64